### PR TITLE
Added custom video player

### PR DESCRIPTION
**Pull request by FireQueen-3010**
## Related Issue
Made a Custom Video player using Javascript

Closes: #[#463]

### Describe the changes you've made
I added the necessary javascript, html and css files corresponding to the project and tested them.

### Describe if there is any unusual behaviour of your code(Write `NA` if there isn't)
NA

### Checklist:

- [X] My code follows the style guidelines of this project.
- [X] I have performed a self-review of my own code.
- [X] I have commented my code, particularly in hard-to-understand areas.
- [X] I have made corresponding changes to the documentation.
- [X] New and existing unit tests pass locally with my changes.

### Screenshots
Put any screenshot(s) of the project here.
image

### CODE REVIEW

**Comment by FireQueen-3010** [comment]
Please review my pr @ridsuteri

**Review by ridsuteri — CHANGES_REQUESTED** [review]
@FireQueen-3010 kindly add detials of this project in the script.js file

**Comment by FireQueen-3010** [comment]
Ok

**Comment by FireQueen-3010** [comment]
Made the changes please review @ridsuteri

**Review by ridsuteri — CHANGES_REQUESTED** [review]
your PR contains some commits which are breaking Expanding card project
kindly fix that

File: public/ExpandingCard.html [hunk 1]
```diff
@@ -0,0 +1,30 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel = "stylesheet" href="../assets/css/Expanding card.css">
```
Comment (ridsuteri) [inline public/ExpandingCard.html:6]: Css file not linked properly 
make it ExpandingCard.css

File: public/ExpandingCard.html [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+        </div>    
+        <div class="panel" style="background-image: url('https://images.unsplash.com/photo-1551009175-8a68da93d5f9?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=1351&q=80')">
+            <h3>City on Winter</h3>
+        </div>
+        <div class="panel" style="background-image: url('https://images.unsplash.com/photo-1549880338-65ddcdfd017b?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=1350&q=80')">
+            <h3>Mountains - Clouds</h3>
+        </div>
+    </div>
+
+    <script src="../assets/js/Expanding card.js"></script>
```
Comment (ridsuteri) [inline public/ExpandingCard.html:28]: js file not linked properly
make it ExpandingCard.js

**Review by ridsuteri — CHANGES_REQUESTED** [review]
these are the changes required

**Comment by FireQueen-3010** [comment]
ok

**Comment by FireQueen-3010** [comment]
Done @ridsuteri

**Review by ridsuteri — APPROVED** [review]
great work @FireQueen-3010 ,thanks for contributing :)

**Comment by FireQueen-3010** [comment]
Thank you @ridsuteri . Please add gssoc and level tags else I cannot receive my pts :)

**Pull request merged**